### DISCUSSION TITLE
I just combined one of my Stable Diffusion safetensors with the video safetensors from text-to-video-ms-1.7b

### DISCUSSION THREAD

**Discussion by ourladyofagi** [deforum-art/sd-webui-text2video#46] [General]
It didn't break it, and I'm not entirely sure what I was expecting, but the output is... good?

(I know nothing about how safetensors or made, what they consist of, or what combining them does, and I just wanted to see what would happen. Added a 70/30 mix with the txt2vid and my... octopus?... model, and... yeah.)

Just thought I'd throw it out there. Not quite sure where else to post something like this haha

(The auto1111 extension is working well. Thanks!)

**Comment by hithereai** [comment]
Give us a link to the model?

**Comment by kabachuha** [comment]
**Absolutely**, post it on [r/StableDiffusion](https://reddit.com/r/stablediffusion). They *love* such experiments

**Comment by ourladyofagi** [comment]
https://user-images.githubusercontent.com/126371399/227062986-7f7202c3-5bb9-496e-a727-398f265eed8b.mp4

**Comment by jparmstr** [comment]
I did some experimenting with this and wasn't able to find anything interesting.

I created 3 different merges between text2video_pytorch_model.pth and Deliberate v2. The .pth file is in CKPT format, not Safetensors. I used Weighted Sum, Don't Copy Config, Bake In VAE: None. For sampling, the prompt is "a raccoon walking in the forest, DSLR, detailed", negative prompt: "blurry, drawing", 30 steps, CFG 12.5, 30 frames, seed 1, 256 x 256, video is 5 FPS, and CRF is 12. The resulting models have different checksums from each other, and are all 5.25 GB like the original model.

https://user-images.githubusercontent.com/33569918/227389067-3eafdb52-312b-4b61-ad34-4802ca2004fd.mp4
Original text2video_pytorch_model

https://user-images.githubusercontent.com/33569918/227389129-6b0e7370-1ff2-457e-acb3-bfac6dd99e6a.mp4
30% Deliberate v2, 70% original model

https://user-images.githubusercontent.com/33569918/227389153-870924c5-af6a-469b-ac23-f212d1ac931a.mp4
70% Deliberate v2, 30% original model

https://user-images.githubusercontent.com/33569918/227389180-74013056-f139-40d8-9ceb-ef2971e38c46.mp4
100% Deliberate v2, 0% original model

The videos are not technically identical (they have different checksums), but they are nearly visually identical. I used Photoshop to compare the first frame from the videos created by the merged models to the first frame from the video created by the original model. I used the "Difference" color blending mode:

![difference 30-70](https://user-images.githubusercontent.com/33569918/227389551-e36bad8d-176c-4e5d-b1c4-920c8b4d0c79.png)
Difference between 30-70 model and original model's first frame

![difference 70-30](https://user-images.githubusercontent.com/33569918/227389592-853da816-01af-455b-b6aa-b48009261919.png)
Difference between 70-30 model and original model's first frame

![difference 0-100](https://user-images.githubusercontent.com/33569918/227389620-ee9dd7fe-f4a5-4490-aac3-041ebdfb5ac0.png)
Difference between 0-100 model and original model's first frame

As you can see there is no visible difference between the frames. These images aren't pure black; there are random pixels with values like "010000" instead of "000000". But they are basically identical.

![brightness example](https://user-images.githubusercontent.com/33569918/227390459-25bc5640-1f71-4c37-a75c-321d8705b8e3.png)
This is an example of what you would see if the images differed from each other. I boosted the Brightness of one of the frames, and this is the difference between that frame and the original one. This is to demonstrate that my method for comparing images works.

**Reply by kabachuha** [reply]
I guess, webui's merger tries to combine weights with names from the sd config, which are likely not present in this model's dictionary

**Reply by ourladyofagi** [reply]
Excellent work. Thanks for getting to the bottom of this.
